dev/gen_version.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# This file should be called from the top-level directory of Coq!
+export LC_ALL=C
+COMPILE_DATE=`date`
+
+# default coq version
+COQ_VERSION=8.8+alpha
```

Comment:
It seems poor that this appears both in `configure.ml` and `dev/gen_version.sh`...

Reply:
I don't want to make `coqversion.ml` to depend on `coq_config.ml`, due to the crazy way it is handled wrt to the `libs`, `check.cma`, etc... We can however have it read from a top-level `VERSION` text file if you think it is better.